lib/sgtn-client/sgtn-client.rb

```diff
@@ -4,6 +4,9 @@
 module SgtnClient
       LOGFILE_SHIFT_AGE = 4
 
+      module Common
+            autoload :BundleID,     "sgtn-client/common/data"
+      end
```

Comment:
This uses only the locale and component. Should we consider adding product name and version ID as well if in case we want to support being able to load translation for multiple VIP "products" in one application?

Reply:
Currently our ruby client doesn't support multiple products. Supporting it will need a big change.
Our interfaces need to add product and version as arguments.

Reply:
Okay, will think more about it later on.